CMakeLists.txt

```diff
@@ -282,19 +289,15 @@ if (ENABLE_LTO AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     message(WARNING "LLD does not work with GCC's LTO object format, switching to Gold.")
     set(BUILD_LINK_WITH_LLD OFF)
   endif ()
-  add_cxx_compiler_option  ("-flto")
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8.0)
```

Comment:
No need to worry about as GCC 9 or newer is required by p4c anyway

Reply:
True, I may as well remove this check...